client_src/js/printerManagementRunner.js

```diff
@@ -3,7 +3,7 @@ import OctoPrintClient from "./lib/octoprint.js";
 import OctoFarmClient from "./lib/octofarm.js";
 import UI from "./lib/functions/ui.js";
 import PrinterManager from "./lib/modules/printerManager.js";
-import PrinterSettings from "./lib/modules/printerSettings.js";
+import { updatePrinterSettingsModal } from "./lib/modules/printerSettings.js";
```

Comment:
Rather than importing the full class I decided to just create a function that applied the state and utilised the statics in the file.